spec/services/search/google/search_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Search::Google::SearchService, type: :service do
+  describe '#call' do
+    context 'given the search result is valid' do
+      it 'returns search page body', vcr: 'search_service/keyword_ruby' do
+        search_result = described_class.new('ruby').call
+
+        expect(search_result).to include('<title>ruby - Google Search</title>')
+      end
+    end
+
+    context 'given the search result is invalid' do
```

Comment:
Sorry, this might not be on the Compass, but for invalid cases, we often uppercase the `INVALID` text:
```suggestion
    context 'given an INVALID search result' do
```

Reply:
No, we usually apply it for words like `invalid`/`no` to emphasize that these are unusual cases 🙌

Reply:
@hoangmirs 🙆‍♂️  Thanks for clarifying 🚀